lara-typescript/src/interactive-api-client/api.spec.ts

```diff
@@ -89,6 +90,23 @@ describe("api", () => {
     api.removeCustomMessageListener();
   });
 
+  it("supports content decoration", () => {
+    const callback = jest.fn();
+    api.addDecorateContentListener(callback);
+    const content: ITextDecorationInfo = {
+      listenerTypes: [{ type: "type" }],
+      words: [],
+      replace: "",
+      wordClass: "word"
+    };
+    mockedPhone.fakeServerMessage({type: "decorateContent", content});
+    api.postDecoratedContentEvent({ type: "type", text: "text" });
+    expect(callback).toHaveBeenCalledTimes(1);
+    api.removeDecorateContentListener();
+    api.postDecoratedContentEvent({ type: "type", text: "text" });
+    expect(callback).toHaveBeenCalledTimes(1);
```

Comment:
This seems to be documenting broken behavior. If the listener is removed wouldn't the callback be expected to not be called?
I think it'd be fine to keep this, but adding a comment explaining it would be helpful.

Reply:
We were already called once before the removal, so the test is verifying that we weren't called again after the removal. I added a comment to clarify.